dspace-spring-rest/src/test/java/org/dspace/app/rest/model/RootRestTest.java

```diff
@@ -0,0 +1,76 @@
+package org.dspace.app.rest.model;
+
+import org.dspace.app.rest.RootRestResourceController;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+
+/**
+ * Created by raf on 26/09/2017.
+ */
+public class RootRestTest {
```

Comment:
You only need to test classes which contain some "complex logic" -> Not only getters and setters.

Please remove this class.